### ISSUE TITLE
No package matching 'pdns-server-0.0.1895...'

### ISSUE THREAD

**Issue by asosso** [PowerDNS/pdns-ansible#36]
Hello,

I'm getting this error with v0.1.1:

```
FAILED! => {"changed": false, "failed": true, "msg": "No package matching 'pdns-server-0.0.1895ge575780-1pdns.xenial\n' is available"}
```

Probably there is a dash to change: https://github.com/PowerDNS/pdns-ansible/commit/2730556e5c3f59d1fabff9e6f8022e764933cf77#diff-2444ad0870f91f17ca6c2a5e96b26823R23 
* from: `pdns_package_name: "{{ pdns_package_name }}-{{ pdns_package_version }}"`
* to: `pdns_package_name: "{{ pdns_package_name }}={{ pdns_package_version }}"`

Anyway isn't also working with =

```
FAILED! => {"cache_update_time": 1508235319, "cache_updated": false, "changed": false, "failed": true, "msg": "'/usr/bin/apt-get -y -o \"Dpkg::Options::=--force-confdef\" -o \"Dpkg::Options::=--force-confold\"    --simulate install 'pdns-server=0.0.1895ge575780-1pdns.xenial\n'' failed: E: Version '0.0.1895ge575780-1pdns.xenial\n' for 'pdns-server' was not found\n", "rc": 100, "stderr": "E: Version '0.0.1895ge575780-1pdns.xenial\n' for 'pdns-server' was not found\n", "stderr_lines": ["E: Version ‘0.0.1895ge575780-1pdns.xenial”, "' for 'pdns-server' was not found"], "stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\n", "stdout_lines": ["Reading package lists...", "Building dependency tree...", "Reading state information..."]}
```

Version 0.0.1895ge575780 it's already installed

```
$ dpkg -l | grep pdns-server
ii  pdns-server                          0.0.1895ge575780-1pdns.xenial              amd64        extremely powerful and versatile nameserver
```

Do you have any idea?

Thank you

**Comment by atosatto** [comment]
Ehi @asosso, can you post here the variables you're passing to the `pdns` role?

**Comment by pieterlexis** [comment]
> * from: pdns_package_name: "{{ pdns_package_name }}-{{ pdns_package_version }}"
> * to: pdns_package_name: "{{ pdns_package_name }}={{ pdns_package_version }}"

This could be the difference between RHEL and Debian

**Comment by asosso** [comment]
Hi @atosatto, here the variables:

```
pdns_install_repo: "{{ pdns_auth_powerdns_repo_master }}"
pdns_config: (with many parameters)
```

@pieterlexis great! you got it 👍

**Comment by atosatto** [comment]
So the issues are actually two:

1. On Debian systems we must use the `=` to separate the package-name and the version
2. There is a trailing `\n` in the package-version when inferred with the `atosatto.packages-extras` modules

**Comment by asosso** [comment]
@atosatto of course. Thank you

**Comment by pieterlexis** [comment]
Should be fixed with #37
